### PR TITLE
Update monero-core_hr.ts

### PR DESCRIPTION
**Pull request by TasmaniaKrama**
This is my first fork/pull request ever on github so I hope I didn't screw anything up.  
I  translated few things and I hope I will continue to translate the whole thing in the future.

### CODE REVIEW

**Comment by TasmaniaKrama** [comment]
This is my first fork/pull request ever on github so I hope I didn't screw anything up.  
I  translated few things and I hope I will continue to translate the whole thing in the future.

File: translations/monero-core_hr.ts [hunk 1]
```diff
@@ -6,106 +6,106 @@
...
+        <translation>Identifikacija plaćanja</translation>
     </message>
 </context>
 <context>
     <name>BasicPanel</name>
     <message>
         <location filename="../BasicPanel.qml" line="97"/>
         <source>Locked Balance:</source>
-        <translation type="unfinished"></translation>
+        <translation>Zaključano Stanje</translation>
```
Comment (erciccione) [inline translations/monero-core_hr.ts]: Please add `:` at the end, like in the source string

File: translations/monero-core_hr.ts [hunk 2]
```diff
@@ -6,106 +6,106 @@
...
         <location filename="../BasicPanel.qml" line="110"/>
         <source>78.9239845</source>
-        <translation type="unfinished"></translation>
+        <translation>78.9239845</translation>
     </message>
     <message>
         <location filename="../BasicPanel.qml" line="133"/>
         <source>Available Balance:</source>
-        <translation type="unfinished"></translation>
+        <translation>Dostupno Stanje</translation>
```
Comment (erciccione) [inline translations/monero-core_hr.ts]: missing `:` as in comment above

**Comment by erciccione** [comment]
ping

**Comment by erciccione** [comment]
@luigi1111 This can be closed

**Pull request closed without merging**